File: app/src/main/java/io/baff/sample/MainActivity.java
package io.baff.sample;
import android.app.Activity;
import android.os.Bundle;
import io.baff.core.GetBaff;
public class MainActivity extends Activity
{
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        GetBaff.ShowGetBaff( this);
    }
}

File: kotlin/app/src/main/java/io/baff/sample/MainActivity.kt
package io.baff.sample

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import io.baff.core.GetBaff

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onRunGetBaff(view: View?) {
        GetBaff.ShowGetBaff(this)
    }

    fun onStartSub(view: View?) {
        val intent = Intent(this, SubActivity::class.java)
        intent.flags = Intent.FLAG_ACTIVITY_REORDER_TO_FRONT
        this.startActivity(intent)
    }
}